pkg/csource/csource.go

```diff
@@ -685,6 +685,9 @@ func (ctx *context) postProcess(result []byte) []byte {
 	result = regexp.MustCompile(`\t*exitf\((.*\n)*?.*\);\n`).ReplaceAll(result, []byte("\texit(1);\n"))
 	result = regexp.MustCompile(`\t*fail(msg)?\((.*\n)*?.*\);\n`).ReplaceAll(result, []byte("\texit(1);\n"))
 
+	// Remove executor include guards.
+	result = regexp.MustCompile(`#define\s+EXECUTOR_.*?_H.*\n`).ReplaceAll(result, nil)
```

Comment:
Do we need another `.*` before `\n`? It looks dangerous. May cause future surprises and troubles.